### DISCUSSION TITLE
Hola!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19316] [GitHub Education] [answered]
Como puedo obtener los beneficios de estudiante en github?

**Comment by Josephglz** [comment, marked as answer]
Existe la página de Github Education https://education.github.com dónde podrás realizar tu solicitud para recibir los beneficios de alumno o en su caso de maestro, dependiendo tu caso, despues de realizar la solicitud, tendrás que esperar a que revisen tu solicitud para recibir dichos beneficios.

**Reply by ghost** [reply]
En que parte debería de registrarme?

**Reply by Josephglz** [reply]
En este link: https://education.github.com/students

**Reply by ghost** [reply]
Muchas gracias!
